src/instruments/src/EFB/Settings/ThrottleConfig/BaseThrottleConfig.tsx

```diff
@@ -15,30 +15,34 @@ interface Props {
     setInitialize,
 }
 
+export const ThrottleContext = React.createContext<{throttlePos: number}>({ throttlePos: 0 });
```

Comment:
It's one value, let's just get the LVar locally.

Reply:
you are completely right, i will fix that soon

Reply:
I removed the usage of the context, again I am very sorry about that!